### PR TITLE
flamenco, vm: add text_sz field to vm_ctx

### PR DESCRIPTION
**Pull request by ravyu-jump**
This PR just propagates the length of the text section to `vm_ctx` for use in `fd_vm_validate`

The following structs are touched as a result:

```mermaid
---
Passing text_sz
---
flowchart TD
	in[/input\ncontaining text size/]
	info[fd_sbpf_elf_info_t]
	prog[fd_sbpf_prog_t]
	val_prog[fd_sbpf_validated_program_t]
	vm[\fd_vm_t\]
	in -- fd_sbpf_elf_peek --> info --> prog --> val_prog & vm
	val_prog --> vm
```

### CODE REVIEW

File: src/app/fdctl/version.h
Comment (ripatel-fd) [inline src/app/fdctl/version.h:1]: This change seems unrelated

File: src/ballet/sbpf/fd_sbpf_loader.c [hunk 1]
```diff
@@ -428,7 +428,10 @@ fd_sbpf_load_shdrs( fd_sbpf_elf_info_t *  info,
            separately done in the sBPF verifier. */
 
   info->text_off = (uint)shdr_text->sh_offset;
-  info->text_cnt = (uint)shdr_get_loaded_size( shdr_text ) / 8U;
+  ulong text_size = shdr_get_loaded_size( shdr_text );
```
Comment (ripatel-fd) [inline src/ballet/sbpf/fd_sbpf_loader.c:431]: Can this be unaligned?

**Review by ripatel-fd — CHANGES_REQUESTED** [review]

File: src/ballet/sbpf/fd_sbpf_loader.c (hunk 1 above)
Comment (ravyu-jump) [reply src/ballet/sbpf/fd_sbpf_loader.c:431]: Alignment checks are done later on in `fd_vm_validate`

Comment (ravyu-jump) [reply src/ballet/sbpf/fd_sbpf_loader.c:431]: Agave allows text section size to be unaligned during ELF loading

File: src/app/fdctl/version.h
Comment (ravyu-jump) [reply src/app/fdctl/version.h:1]: Yeah, it breaks CI though. Shouldn't have been tracked in the first place

File: src/ballet/sbpf/fd_sbpf_loader.c (hunk 1 above)
Comment (ripatel-fd) [reply src/ballet/sbpf/fd_sbpf_loader.c:431]: Thanks. Scary! :)

**Review by topointon-jump — APPROVED** [review]

**Review by ripatel-fd — APPROVED** [review]
